tests/filters/test_filters.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import numpy as np
+from scipy.sparse import coo_matrix
+
+from ophys_etl.filters.filters import filter_longest_edge_length
+
+
+@pytest.mark.parametrize("coo_rois, longest_edge_thrsh, expected_rois",
+                         [([coo_matrix(np.array([[0, 0, 0, 0, 0],
```

Comment:
I'll say this is ok for now given the simplicity of the function. In the future when you are building test cases for functions that operate on lists you should make sure to include the common list cases -- empty list, singleton list, and 2+ length list. These are all singleton lists.

Reply:
Added a test case for empty list and multi element list with elements that will and won't pass the filter.

Reply:
There's another test case that you're missing that I believe will reveal a bug -- set the longest edge of the array equal to the threshold.

Format this case appropriately and try:
edge threshold = 3
array:
```
[[1, 1, 1]]
```

Reply:
Great find, I found the bug in the function, fixed it, and added tests for this case!